filters/auth/tokeninfo.go

```diff
@@ -218,6 +240,18 @@ func (s *tokeninfoSpec) CreateFilter(args []interface{}) (filters.Filter, error)
 		return nil, filters.ErrInvalidFilterParameters
 	}
 
+	if s.typ == checkOAuthTokeninfo {
+		if len(sargs) != 1 {
+			return nil, fmt.Errorf("requires single string argument")
```

Comment:
maybe add the filter name

Reply:
> maybe add the filter name to the error message

https://github.com/zalando/skipper/pull/1908